mavedb_mapping/main.py

```diff
@@ -0,0 +1,13 @@
+from metadata_process import metadata_obtain
+from blat_alignment import mave_to_blat
+from transcript_selection import main
+from vrs_mapping import vrs_mapping
+from time import time
+
+
+def main_map(urn):
```

Comment:
consider using pytest fixtures instead; there MAY be package-level fixtures, otherwise consider creating fixture modules (like `main.py` here) that can be imported.